Forge.TreeWalker/src/TreeWalkerSession.cs

```diff
@@ -138,6 +138,11 @@ public class TreeWalkerSession : ITreeSession
         /// </summary>
         private bool hasSessionRehydrated;
 
+        /// <summary>
+        /// The context value whether the actions in this TreeNodes are skipped.
+        /// </summary>
+        private string currentNodeSkipActionContext;
```

Comment:
>currentNodeSkipActionContext [](start = 23, length = 28)

Let's use a similar summary as the get method.

/// The string context if the actions in the current TreeNode were skipped, or null if actions were not skipped.

Reply:
updated

---
In reply to: [505915501](https://github.com/microsoft/Forge/pull/32#discussion_r505915501) [](ancestors = 505915501)